src/Sylius/Component/Promotion/Model/CatalogPromotion.php

```diff
@@ -155,4 +157,24 @@ public function removeAction(CatalogPromotionActionInterface $action): void
         $action->setCatalogPromotion(null);
         $this->actions->removeElement($action);
     }
+
+    public function isEnabled(): bool
+    {
+        return $this->enabled;
+    }
+
+    public function setEnabled(?bool $enabled): void
+    {
+        $this->enabled = $enabled;
+    }
+
+    public function enable(): void
+    {
+        $this->setEnabled(true);
+    }
+
+    public function disable(): void
+    {
+        $this->setEnabled(false);
+    }
```

Comment:
Do we really need these functions (imo looks useless but I might be wrong)? They are just setEnabled in other form.

Reply:
these functions are from `ToggleableInterface` (interface from Sylius) it is not my invention :D